src/graphql/typeDefs/mutations/venueRooms.graphql

```diff
@@ -0,0 +1,8 @@
+type VenueRoomsMutation {
+  "Create a new Venue Room"
+  create(venueRoom: VenueRoomCreateInput!): VenueRoom! @auth(requires: "venues")
```

Comment:
why not just admin? do we really need another role?

Reply:
The role was already there, I used the same role which was on Venue currently.

Reply:
Did we ever really create it? Or better said now that we're here is it really needed any longer? Or is it just over completing things?

Reply:
I see no use for it. My assumption is that we added it as others may make venue changes. I planned to use admin and when I saw that the other venue  mutations used, 'venue', I used it. It's a good opportunity to remove it. If we decide to, I'll remove it from auth0 too.